.github/workflows/validate.yml

```diff
@@ -58,8 +60,7 @@ jobs:
     runs-on: ubuntu-latest
     if:
       ${{ github.repository == 'testing-library/react-testing-library' &&
-      contains('refs/heads/main,refs/heads/beta,refs/heads/next,refs/heads/alpha',
```

Comment:
Got introduced in https://github.com/testing-library/dom-testing-library/pull/809 but at that point we didn't have a a filter in `on.push.branches` so this seems redundant now.